.circleci/config.yml

```diff
@@ -0,0 +1,128 @@
+version: 2.1
+
+orbs:
+  gem: doximity/gem-publisher@0
+
+executors:
+  ruby-latest:
+    resource_class: small
+    docker:
+      - image: circleci/ruby:latest
+        environment:
+          BUNDLE_VERSION: "~> 1.17"
+
+# yaml anchor filters
+master_only: &master_only
+  filters:
+    branches:
+      only: master
+    tags:
+      ignore: /.*/
+pr_only: &pr_only
+  filters:
+    branches:
+      ignore: master
+    tags:
+      ignore: /.*/
+version_tags_only: &version_tags_only
+  filters:
+    branches:
+      ignore: /.*/
+    tags:
+      only: /^v.*/
+
+jobs:
+  build:
+    executor: ruby-latest
+    steps:
+      - checkout
+      - run:
+          name: Install Bundler specific version
+          command: |
+            gem install bundler --version "${BUNDLE_VERSION}" --force
+      - restore_cache:
+          keys:
+            - v1-bundle-{{ checksum "Gemfile.lock" }}-
+      - run:
```

Comment:
Should we `bundle clean` here as well? From the Circle docs: https://circleci.com/docs/2.0/caching/#bundler-ruby

Reply:
Nope, ignore me. That's for partial dependency caching. Nothing to see here :D

Reply:
Maybe it's still a good idea?